aggregator/models.py

```diff
@@ -218,6 +220,9 @@ class LocalDjangoCommunity(models.Model):
     )
     is_active = models.BooleanField(default=True)
     created_at = models.DateTimeField(auto_now_add=True)
+    approval_status = models.CharField(
+        max_length=1, choices=STATUS_CHOICES, default=PENDING_FEED
```

Comment:
If `PENDING_FEED` constant is not used only for Feed maybe we could rename it to sth like `PENDING_STATUS` or just `PENDING`